sql/database.go

```diff
@@ -30,6 +28,9 @@ var (
 	ErrObjectExists = errors.New("database: object exists")
 	// ErrTooNew is returned if database version is newer than expected.
 	ErrTooNew = errors.New("database version is too new")
+	// ErrOld is returned when the database version differs from the expected one
+	// and migrations are disabled.
+	ErrOld = errors.New("old database version")
```

Comment:
NIT: naming could be better

```suggestion
	// ErrOldSchema is returned when the database version differs from the expected one
	// and migrations are disabled.
	ErrOldSchema = errors.New("old database version")
```

Reply:
Done